### ISSUE TITLE
Support external monitors

### ISSUE THREAD

**Issue by AlexPerathoner** [AlexPerathoner/SlimHUD#82]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Is your feature request related to a problem? Please describe

External monitors whose brightness can't be changed directly from the OS aren't recognised by SlimHUD. Changing their brightness doesn't show the HUD.

### Describe the solution you'd like

Use the implementation used in [MonitorControl](https://github.com/MonitorControl/MonitorControl) to retrieve the brightness

### Describe alternatives you've considered

Don't do anything

**Comment by Ein-Tim** [comment]
@AlexPerathoner I just started using this great app today and this is the only feature I'm missing! Besides of that, great work!

**Comment by AlexPerathoner** [comment]
Ohh a fellow Münchner :)

**Comment by Ein-Tim** [comment]
Haha, yeah! I'm waving over to you at the TUM! 👋

Are there any plans to support external monitors soon?

**Comment by AlexPerathoner** [comment]
Yes, but I haven't looked into it yet. So it will probably take a couple weeks until I have a release ready.
Anyway it's in top priority now :)

**Comment by usamasaleem1** [comment]
any updates?

**Comment by sup3rb3ar** [comment]
Just want to note that [BetterDisplay](https://betterdisplay.pro) (successor to MonitorControl) does support integration with external OSD presentation apps.

https://github.com/waydabber/BetterDisplay/wiki/Integration-features,-CLI#osd-notification-dispatch-integration

The API seems to be developed for MediaMate but made in a generic way so it's open to other apps as well.
